swarm/network/stream/stream.go

```diff
@@ -146,7 +146,7 @@ func NewRegistry(addr *network.BzzAddr, delivery *Delivery, db *storage.DBAPI, i
 				timer := time.NewTimer(options.SyncUpdateDelay)
 				// Hard limit to sync update delay, preventing long delays
 				// on a very dynamic network
-				maxTimer := time.NewTimer(3 * time.Minute)
+				maxTimer := time.NewTimer(0 * time.Minute)
```

Comment:
Why is the timer set to 0 in production code? If there is a need for testing to be 0, then it should be configurable, or settable globally just for running the test. If it is a problem for production code and functionality it should probably removed.

Reply:
I have no idea how this got changed. I will change it back to 3